doc/meetings/project-meeting-2018-06-19.md

```diff
@@ -0,0 +1,69 @@
+# Build WG: Discuss new projects #1330
```

Comment:
Are you able to make the header / structure of the minutes match the normal WG minutes (See https://github.com/nodejs/build/blob/master/doc/meetings/build-wg-meeting-2018-06-12.md as an example)